### ISSUE TITLE
Error launching on NextJS

### ISSUE THREAD

**Issue by Gr33nLight** [wallabyjs/console-ninja#40]
I'm using version 0.0.28 and NextJS 12.3.1
I added a console.log at the start of a page and launching the app in development gets me this error that prevent the app from running, note that before this error happens, I get the console.log in the editor as expected. Removing the console.log and restarting the dev command solves the error

```
error - Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.
    at renderElement (/Users/demo/Documents/Dev/JS/ewi-frontend/node_modules/react-dom/cjs/react-dom-server.browser.development.js:6043:9)
    at renderNodeDestructiveImpl (/Users/demo/Documents/Dev/JS/ewi-frontend/node_modules/react-dom/cjs/react-dom-server.browser.development.js:6104:11)
    at renderNodeDestructive (/Users/demo/Documents/Dev/JS/ewi-frontend/node_modules/react-dom/cjs/react-dom-server.browser.development.js:6076:14)
    at renderNode (/Users/demo/Documents/Dev/JS/ewi-frontend/node_modules/react-dom/cjs/react-dom-server.browser.development.js:6259:12)
    at renderHostElement (/Users/demo/Documents/Dev/JS/ewi-frontend/node_modules/react-dom/cjs/react-dom-server.browser.development.js:5642:3)
    at renderElement (/Users/demo/Documents/Dev/JS/ewi-frontend/node_modules/react-dom/cjs/react-dom-server.browser.development.js:5952:5)
    at renderNodeDestructiveImpl (/Users/demo/Documents/Dev/JS/ewi-frontend/node_modules/react-dom/cjs/react-dom-server.browser.development.js:6104:11)
    at renderNodeDestructive (/Users/demo/Documents/Dev/JS/ewi-frontend/node_modules/react-dom/cjs/react-dom-server.browser.development.js:6076:14)
    at renderNode (/Users/demo/Documents/Dev/JS/ewi-frontend/node_modules/react-dom/cjs/react-dom-server.browser.development.js:6259:12)
    at renderChildrenArray (/Users/demo/Documents/Dev/JS/ewi-frontend/node_modules/react-dom/cjs/react-dom-server.browser.development.js:6211:7) {
  page: '/auth'
}

```

**Comment by Gr33nLight** [comment]
This is the code change in question

```
export default function Auth() {
  console.log('a');
  return (
    <div className="flex flex-1 bg-stone-50">
     ...
    </div>
  );
}

```

**Comment by ArtemGovorov** [comment]
We've tried a freshly generated Next 12 app, created `auth.js` file with the exact same content, but everything is working for us as expected with no errors:

![cn2gif](https://user-images.githubusercontent.com/979966/206990695-76aa6032-8622-47f9-bbd8-70b02b7525fa.gif)

Can you please create a sample repo demonstrating the issue, or maybe you can see something in the recording that we're missing to reproduce it?

**Comment by ArtemGovorov** [comment]
@Ahmadh26 

Re: `Module parse failed: Identifier '_react' has already been declared (19:6)`, thanks for providing the details,  we were able to reproduce the issue.

Re: `error - Error: No QueryClient set, use QueryClientProvider to set one` - can you please share the code of the page that is casing the issue?

**Comment by Ahmadh26** [comment]
@ArtemGovorov 

it is just a simple useQuery in a react component. i cannot share the full code in public but here is some: 

```tsx 
import { useRef, useState } from 'react'
import { useRouter } from 'next/router'
import { fetchAllCountries, fetchCategories } from '@lib/queries'
import { useQueries, useQuery } from '@tanstack/react-query'

type Props = {
  searchWrapper?: string
  lineWrapper?: string
  isSearchPage?: boolean
}

export default function SearchComponent(props: Props) {
  const { searchWrapper, lineWrapper } = props
  const router = useRouter()
  const keywordRef = useRef<HTMLInputElement>(null)

  const [address, setAddress] = useState<string>('')

  const [search, setSearch] = useState<{
    cat?: number
    subCat?: number
    key?: string
    location?: string
    dist?: number
    country?: string
    title?: number
  }>({
    cat: 0,
    subCat: 0,
    key: '',
    location: '',
    dist: 0,
    country: '',
    title: 0,
  })

  // const results = useQueries({
  //   queries: [
  //     {
  //       queryKey: ['countries'],
  //       queryFn: () => fetchAllCountries(),
  //       staleTime: Infinity,
  //     },
  //     {
  //       queryKey: ['categories'],
  //       queryFn: () => fetchCategories(),
  //       staleTime: Infinity,
  //     },
  //   ],
  // })

  const { data: countriesData, isLoading: cLoading } = useQuery({
    queryKey: ['countries'],
    queryFn: () => fetchAllCountries(),
    staleTime: Infinity,
  })
  const { data: catData, isLoading } = useQuery({
    queryKey: ['categories'],
    queryFn: () => fetchCategories(),
    staleTime: Infinity,
  })

  const categories = catData ?? []
  const countries = countriesData ?? []
  const subCategory =
    categories.find((cat) => cat.id === search.cat)?.subcategories ?? []
  const searchOptions = {
    componentRestrictions: {
      country:
        countries.find((x) => x.name === search.country)?.isoAlpha2 ?? 'au',
    },
  }

  return (
  <div>some stuff here</div>
  )
}


```

This is a component i have in my homepage. it is the one causing an issue. unfortunately i cannot share more code publicly as this is my work code. 

I am more than happy to share more in private  (discord if you want)

**Comment by ArtemGovorov** [comment]
@Ahmadh26 Thanks! 
When exactly does the `Error: No QueryClient set` show up:
- when you add some console.log anywhere in that file?
- or it's starting to show when you just edit the file with Console Ninja running?
- or it's starting to show when after you uninstall Console Ninja (before you `remove .next folder and restart the dev server`)?

**Comment by ArtemGovorov** [comment]
@Ahmadh26 The `Module parse failed: Identifier '_react' has already been declared (19:6)` error should be fixed in the v0.0.31+. After the version update, please stop the Next.js CLI process, please delete the `.next` folder and run `npm run dev` (only required once). After doing that, please check if the other 2 reported errors ever show up, and if they do - please let us know.

**Comment by Ahmadh26** [comment]
@ArtemGovorov Both issues are resolved!!! 
i was creating a video showing you what's happening and out of nowhere it didnt happen. realized that the extension updated lol!

Thank you for the quick support :)! i'll let you know if something else comes up!

**Comment by Ahmadh26** [comment]
@ArtemGovorov

one more question, any reason this is happening: 
![image](https://user-images.githubusercontent.com/57562297/207037977-8495e381-e295-43d4-90ed-9fcb843c367d.png)


results are posted in terminal + next to the console.log in the code editor, but not in the split tab.

**Comment by ArtemGovorov** [comment]
@Ahmadh26 Thanks for the report. We can reproduce the issue, we'll be looking into it a few hours.

**Comment by ArtemGovorov** [comment]
The missing details issue should be now fixed in the latest version (0.0.33+).
